src/utils/config.js

```diff
@@ -0,0 +1,23 @@
+const ALLOWED_MARKDOWN_TAGS = [
```

Comment:
I'm guessing this config was brought over from Interviewer? If we aren't actually using `ALLOWED_MARKDOWN_TAGS`, I think we should remove it here.

Reply:
Removed this.